http-server-netty/src/test/groovy/io/micronaut/http/server/netty/context/ServerRequestContextSpec.groovy

```diff
@@ -109,30 +113,30 @@ class ServerRequestContextSpec extends Specification {
         @Get("/method")
         String method() {
             def request = ServerRequestContext.currentRequest().orElseThrow { -> new RuntimeException("no request") }
-            request.uri
```

Comment:
technically a breaking change, but this was a test bug. the methods declare return type String but actually returned URI.